ouroboros-consensus/src/Ouroboros/Consensus/MiniProtocol/ChainSync/Server.hs

```diff
@@ -89,61 +92,85 @@ chainSyncBlocksServer tracer chainDB flr =
 --
 -- All the hard work is done by the 'Follower's provided by the 'ChainDB'.
 --
-chainSyncServerForFollower ::
-     forall m blk b.
-     ( IOLike m
-     , HasHeader (Header blk)
-     )
-  => Tracer m (TraceChainSyncServerEvent blk)
-  -> ChainDB m blk
-  -> Follower  m blk (WithPoint blk b)
-  -> ChainSyncServer b (Point blk) (Tip blk) m ()
-chainSyncServerForFollower tracer chainDB flr =
+chainSyncServerForReader
```

Comment:
Please don't change the signature style. Consensus likes `::` at the end of the line.

Reply:
Ugh. I see now that this module is already inconsistent with its `::` placement. Sorry for the confusion.

We do overall prefer `::` at the end of the line.

Reply:
Fixed it. I tried to follow the guide for old and new signatures in this module.